core/controllers/base_test.py

```diff
@@ -1423,3 +1462,195 @@ def test_that_correct_derived_class_does_not_raise_exception(self):
+            # are tuples. So, below check is to handle them.
+            if isinstance(route, tuple):
+                continue
+            else:
+                handler = route.handler
+            handler_class_name = handler.__name__
+
+            if handler_class_name in self.non_schema_handlers:
+                continue
+
+            handler_request_methods = []
+            if handler.get != base.BaseHandler.get:
+                handler_request_methods.append('GET')
+            if handler.put != base.BaseHandler.put:
+                handler_request_methods.append('PUT')
+            if handler.post != base.BaseHandler.post:
+                handler_request_methods.append('POST')
+            if handler.delete != base.BaseHandler.delete:
+                handler_request_methods.append('DELETE')
+            schema_keys = [
+                key for key in handler.HANDLER_ARGS_SCHEMAS.keys()]
+            missing_schema_keys = (
+                set(handler_request_methods) - set(schema_keys))
+
+            if missing_schema_keys:
+                handlers_with_missing_schema_keys.append(handler_class_name)
+                schema_keys_match_with_hansler_req_methods = False
+                self.log_line('Missing key in HANDLER_ARGS_SCHEMAS for %s: '
+                    '%s. %s' % (handler_class_name, ', '.join(
+                        missing_schema_keys), 'FAIL'))
+
+        self.assertTrue(schema_keys_match_with_hansler_req_methods,
+            'Missing schema keys in HANDLER_ARGS_SCHEMAS '
+                'for [ %s ] classes.' % ', '.join(
+                    handlers_with_missing_schema_keys))
+
+    def test_default_value_in_schema_conforms_with_schema(self):
+        handlers_checked = []
+        schema_validation = True
+        for route in main.URLS:
+            # URLS = MAPREDUCE_HANDLERS + other handlers. MAPREDUCE_HANDLERS
+            # are tuples. So, below check is to handle them.
+            if isinstance(route, tuple):
+                continue
+            else:
+                handler = route.handler
+            handler_class_name = handler.__name__
+
+            if handler_class_name in self.non_schema_handlers:
+                continue
+
+            schemas = handler.HANDLER_ARGS_SCHEMAS
+            for keys, schema in schemas.items():
+                for key, value in schema.items():
+                    if 'default_value' not in value:
+                        continue
+                    default_value = {key: value['default_value']}
+                    default_value_schema = {key: value}
+
+                    errors = payload_validator.validate(
+                        default_value, default_value_schema, True)
+                    if not errors:
+                        continue
+
+                    self.log_line('Handler: %s, argument: %s, default value '
+                        'validation: %s' % (handler_class_name, key, 'FAIL'))
```

Comment:
```suggestion
                    self.log_line(
                        'Handler: %s, argument: %s, default value '
                        'validation: %s' % (handler_class_name, key, 'FAIL'))
```
In general, if there is multiple lines after an opening bracket, then the convention in Oppia is that it should start from a new line.

Reply:
Done